### DISCUSSION TITLE
MQTT settings ohne Benutzer und Passwort

### DISCUSSION THREAD

**Discussion by electric-m** [camueller/SmartApplianceEnabler#177] [Installation und Betrieb]
Hallo zusammen,

mich wollte eben in den Einstellungen meine bereits bestehende Mosquitto Instanz hinterlegen.
Ich scheitere deshalb, weil ich kein Eingabefeld für Benutzer und Passwort habe, sondern nur für die IP und Port.
Authentifizierung am Broker möchte ich nicht deaktivieren.

Edit:
Betrieb unter Docker auf dem Raspberry Pi 3

In der Doku finde ich nichts dazu.
![EEE310EE-E998-4DBD-AB88-749EC8D195B8](https://user-images.githubusercontent.com/40071240/151341096-871d6e02-965c-42d2-9203-c5f7349d173d.jpeg)

**Comment by camueller** [comment]
Aktuell wird die Authentifizierung am MQTT-Broker noch nicht unterstützt. Das kommt aber noch ...

**Comment by electric-m** [comment]
In Ordnung, danke für die Info. Heißt aber, dass ich SAE so aktuell noch nicht nutzen kann, oder?

**Comment by camueller** [comment]
Zumindest nicht mit Deinem vorhanden MQTT-Broker. Du kannst ja temporär einen zweiten Broker laufen lassen.
